examples/Esp8266-http/esp8266_wifi.h

```diff
@@ -124,8 +124,9 @@ void doRequest(WiFiClientSecure* client, boolean isGet, String postData) {
 }
 
 void sendTelemetry(String data) {
+  rbase64.encode(data);
   String postdata =
-      String("{\"binary_data\": \"") + rbase64.encode(data) + String("\"}");
+      String("{\"binary_data\": \"") + rbase64.result() + String("\"}");
```

Comment:
I'm seeing:

```
/var/folders/w8/y0hz57jn5pb65ptmyfs_dy1w0055ky/T/arduino_modified_sketch_636619/Esp32-ssd1306.ino: In function 'void sendTelemetry(String)':
Esp32-ssd1306:108: error: 'class rBASE64' has no member named 'result'
       String("{\"binary_data\": \"") + rbase64.result() + String("\"}");
```

Which version of rBase64 does this require?

Reply:
The correct use of Base64 encoding is in this PR (Link: https://github.com/boseji/rBASE64).